docs/pages/concepts/networking/components/network-manager.md

```diff
@@ -3,7 +3,7 @@ title: Liqo-Network-Manager
 weight: 1
 ---
 
-The Liqo Network Manager component manages the creation of the`networkconfigs.net.liqo.io` custom resources for the remote clusters. It also processes the`networkconfigs.net.liqo.io` received from the remote clusters remapping the remote networks if they overlap with any of the address spaces used in the local cluster. And when the`networkconfigs.net.liqo.io` exchanged by the two clusters are ready the Liqo Network Manager creates a new resource of type `tunnelendpoints.net.liqo.io` which models the network interconnection between the two clusters. The `tunnelendpoints.net.liqo.io` is reconciled by the [Tunnel Operator](../gateway#tunnel-operator) and the [Route Operator](../route#route-operator).
+The Liqo Network Manager component manages the creation of the`networkconfigs.net.liqo.io` custom resources for the remote clusters. It also processes the`networkconfigs.net.liqo.io` received from the remote clusters remapping the remote networks if they overlap with any of the address spaces used in the local cluster (See [IPAM](#ipam) for more information). And when the`networkconfigs.net.liqo.io` exchanged by the two clusters are ready the Liqo Network Manager creates a new resource of type `tunnelendpoints.net.liqo.io` which models the network interconnection between the two clusters. The `tunnelendpoints.net.liqo.io` is reconciled by the [Tunnel Operator](../gateway#tunnel-operator) and the [Route Operator](../route#route-operator).
 
 The following diagram illustrates how Liqo Network Manager creates the `tunnelendpoints.net.liqo.io`.
```

Comment:
Add a description of the above diagram. The content is not obvious for a reader not familiar with the matter.